chore: update README (#85)
skymavis/waypoint-js: README.md

## Documentation

[Head to the documentation](https://docs.skymavis.com/mavis/ronin-waypoint/reference/web-sdk) to learn about the Ronin Waypoint SDK.

## Packages & Apps

| Name                                                                        | Features                                                                        |
| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
| [packages/waypoint](/packages/waypoint)                                     | SDK lets developers integrate with Ronin Waypoint seamlessly & easily.          |
| [packages/waypoint/headless](/packages/waypoint/headless)                   | SDK lets developers interact with the Ronin Waypoint wallet without a pop-up.   |
| [apps/playground](/apps/playground)                                         | Playground lets developers explore the various features of the Ronin Waypoint.  |
| [apps/example-daily-checkin](/apps/example-daily-checkin)                   | Example of a daily check-in application.                                        |

> | [apps/example-daily-checkin-no-popup](/apps/example-daily-checkin-no-popup) | Example of a daily check-in application run on Telegram that uses headless SDK. |

Reviewer: ```suggestion
| [apps/example-daily-checkin-no-popup](/apps/example-daily-checkin-no-popup) | Example of a daily check-in application that uses headless SDK. (Telegram for example) |
```
